airbyte-webapp/src/scss/_colors.scss

```diff
@@ -1,4 +1,6 @@
 /* stylelint-disable color-no-hex, color-hex-length  */
+$blue-30: #f4f4ff;
+$blue-40: #f0efff;
```

Comment:
:information_source: Took from our color palette in Figma, those where still missing here.